docs/architecture/application.md

```diff
@@ -71,6 +71,18 @@ method;
 app.Features.AddDatabase(c => c.MySql(autoUpdateSchema: true)
 ```
 
+#### Overriding A Configuration
+
+To override the configurations of base features, follow 
+`ConfigurationOverrider` [conventions](../architecture/feature.md#conventions)
+and override configurations in `IFeature` implementation.
```

Comment:
Revise this as follows;

`To override configurations of added features, follow [feature conventions](...), create a ``ConfigurationOverrider`` feature and add your overrider after all other features as shown below;`